Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -400,7 +400,16 @@ namespace O3DE::ProjectManager
                 auto result = PythonBindingsInterface::Get()->CreateGem(templateLocation, m_createGemInfo);
                 if (result.IsSuccess())
                 {
-                    emit CreateButtonPressed();
+                    //make sure the gem is registered, otherwise our changes will not persist
+                    auto registerResult = PythonBindingsInterface::Get()->RegisterGem(m_createGemInfo.m_path);
```

Comment:
CreateGem can register the gem after creating it which would save the second call to PythonBindings/Python. It looks like the registerGem parameter of CreateGem is used as no_register right now though so it's likely doing the opposite of what it should